range.c

```diff
@@ -982,11 +982,15 @@ range_each(VALUE range)
 
 /*
  *  call-seq:
- *     rng.begin    -> obj
+ *    begin -> object
```

Comment:
This is a case where we may want to use `self.begin`, as `begin` is a keyword and you cannot use it as a bare method call.

Reply:
Fixed.